lint/lint.py

```diff
@@ -10,6 +10,7 @@
 import sys
 
 from collections import defaultdict
+from urlparse import urlsplit
```

Comment:
This needs to be `from six.moves.urllib.parse import urlsplit` for Py3 compatibility.